diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/ots-0.5.0/Makefile.am src/ots-0.5.0/Makefile.am
--- origsrc/ots-0.5.0/Makefile.am	2007-04-08 11:15:44.000000000 -0500
+++ src/ots-0.5.0/Makefile.am	2007-08-28 10:55:25.737924300 -0500
@@ -1,5 +1,5 @@
 
-SUBDIRS=src dic doc
+SUBDIRS=src dic # doc
 DIST_SUBDIRS=src articles dic doc
 
 pkgconfigdir = $(libdir)/pkgconfig
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/ots-0.5.0/src/Makefile.am src/ots-0.5.0/src/Makefile.am
--- origsrc/ots-0.5.0/src/Makefile.am	2007-04-08 11:21:51.000000000 -0500
+++ src/ots-0.5.0/src/Makefile.am	2007-08-28 10:48:10.212839500 -0500
@@ -16,7 +16,7 @@
 
 DEFS = @DEFS@ -DVERSION=\"@OTS_VERSION@\" -DDICTIONARY_DIR=\"$(pkgdatadir)/\"
 
-INCLUDES = -I$(top_srcdir) $(OTS_CFLAGS) $(ANSI_CFLAGS)
+INCLUDES = -I$(top_builddir) $(OTS_CFLAGS) $(ANSI_CFLAGS)
 
 lib_LTLIBRARIES = libots-1.la
 
@@ -43,9 +43,7 @@
 	libots.h
 
 bin_PROGRAMS=ots
-ldadd = $(top_builddir)/src/libots-1.la $(OTS_LIBS) $(POPT_LIBS)
-
-ots_LDADD=$(ldadd)
+ots_LDADD = libots-1.la $(OTS_LIBS) $(POPT_LIBS)
 
 install-data-local: install-libtool-import-lib
 
