--- origsrc/ots-0.5.0/Makefile.am	2007-04-08 11:15:44.000000000 -0500
+++ src/ots-0.5.0/Makefile.am	2007-08-28 10:55:25.737924300 -0500
@@ -1,5 +1,5 @@
 
-SUBDIRS=src dic doc
+SUBDIRS=src dic # doc
 DIST_SUBDIRS=src articles dic doc
 
 pkgconfigdir = $(libdir)/pkgconfig
--- origsrc/ots-0.5.0/src/Makefile.am	2007-04-08 11:21:51.000000000 -0500
+++ src/ots-0.5.0/src/Makefile.am	2007-08-28 10:48:10.212839500 -0500
@@ -16,7 +16,7 @@
 
 DEFS = @DEFS@ -DVERSION=\"@OTS_VERSION@\" -DDICTIONARY_DIR=\"$(pkgdatadir)/\"
 
-INCLUDES = -I$(top_srcdir) $(OTS_CFLAGS) $(ANSI_CFLAGS)
+INCLUDES = -I$(top_builddir) $(OTS_CFLAGS) $(ANSI_CFLAGS)
 
 lib_LTLIBRARIES = libots-1.la
 
@@ -43,9 +43,7 @@
 	libots.h
 
 bin_PROGRAMS=ots
-ldadd = $(top_builddir)/src/libots-1.la $(OTS_LIBS) $(POPT_LIBS)
-
-ots_LDADD=$(ldadd)
+ots_LDADD = libots-1.la $(OTS_LIBS) $(POPT_LIBS)
 
 install-data-local: install-libtool-import-lib
 
